cmd/start.go

```diff
@@ -1315,20 +1318,23 @@ func createOrUpdateSuperNodeConfig(ctx context.Context, config *configs.Config)
 		mdlDataPath := filepath.Join(config.WorkingDir, constants.MDLDataDir)
 
 		toolConfig, err := utils.GetServiceConfig(string(constants.SuperNode), configs.SupernodeDefaultConfig, &configs.SuperNodeConfig{
-			LogLevel:      constants.SuperNodeDefaultLogLevel,
-			LogFilePath:   config.Configurer.GetSuperNodeLogFile(config.WorkingDir),
-			SNTempDir:     snTempDirPath,
-			SNWorkDir:     config.WorkingDir,
-			RQDir:         rqWorkDirPath,
-			DDDir:         filepath.Join(config.Configurer.DefaultHomeDir(), constants.DupeDetectionServiceDir),
-			SuperNodePort: portList[constants.SNPort],
-			P2PPort:       portList[constants.P2PPort],
-			P2PDataDir:    p2pDataPath,
-			MDLPort:       portList[constants.MDLPort],
-			RAFTPort:      portList[constants.RAFTPort],
-			MDLDataDir:    mdlDataPath,
-			RaptorqPort:   constants.RRServiceDefaultPort,
-			DDServerPort:  constants.DDServerDefaultPort,
+			LogLevel:                        constants.SuperNodeDefaultLogLevel,
+			LogFilePath:                     config.Configurer.GetSuperNodeLogFile(config.WorkingDir),
+			SNTempDir:                       snTempDirPath,
+			SNWorkDir:                       config.WorkingDir,
+			RQDir:                           rqWorkDirPath,
+			DDDir:                           filepath.Join(config.Configurer.DefaultHomeDir(), constants.DupeDetectionServiceDir),
+			SuperNodePort:                   portList[constants.SNPort],
+			P2PPort:                         portList[constants.P2PPort],
+			P2PDataDir:                      p2pDataPath,
+			MDLPort:                         portList[constants.MDLPort],
+			RAFTPort:                        portList[constants.RAFTPort],
+			MDLDataDir:                      mdlDataPath,
+			RaptorqPort:                     constants.RRServiceDefaultPort,
+			DDServerPort:                    constants.DDServerDefaultPort,
+			NumberOfChallengeReplicas:       constants.NumberOfChallengeReplicas,
+			StorageChallengeExpiredDuration: constants.StorageChallengeExpiredDuration,
+			IsLeader:                        flagMasternodeMetaDBLeader,
```

Comment:
Sorry, no
You can set it up in the config file, but it shouldn't be in the `pastelup`